types/react-star-ratings/index.d.ts

```diff
@@ -0,0 +1,29 @@
+// Type definitions for react-star-ratings 2.3
+// Project: https://github.com/ekeric13/react-star-ratings
+// Definitions by: Andrew Shkurenko <https://github.com/vanillkay>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
```

Comment:
```suggestion
```
it's probably from other package, minimum version tested/supported now is 3.7. No need to hardcode version unless specific reason (e.g TS feature)